### ISSUE TITLE
qgis doesn't see WFS data

### ISSUE THREAD

**Issue by tarikcarli** [3liz/py-qgis-server#14]
I setup py-qgis-server with docker. My docker command is "docker run -p 8000:8080 --restart=always -d -v /home/carli/Tubitak/server/projects:/projects -e QGSRV_SERVER_WORKERS=2 -e QGSRV_LOGGING_LEVEL=DEBUG -e QGSRV_CACHE_ROOTDIR=/projects -e QGSRV_CACHE_SIZE=10 3liz/qgis-map-server:3.10"

I loaded data and qgis project file in volumed directory but I cannot see layer in qgis. Are there any people to help me out there ?
![container folder](https://user-images.githubusercontent.com/63655791/87047929-cc824000-c203-11ea-81fb-099afa717316.png)
![data-folder](https://user-images.githubusercontent.com/63655791/87047943-d146f400-c203-11ea-8a82-0e744b8b30c5.png)
![Screenshot from 2020-07-09 16-43-34](https://user-images.githubusercontent.com/63655791/87047953-d2782100-c203-11ea-8b1e-b1172fe16580.png)
![web-url](https://user-images.githubusercontent.com/63655791/87047956-d310b780-c203-11ea-8a4c-ea26d4250c46.png)

**Comment by dmarteau** [comment]
Please be more specific: is this a Qgis issue or a py-qgis-server issue ?  

You should also check the py-qgis-server logs to see if your layers are ok.

I see two projects in your coastline folder: coastline.qgs and coastline.qgz,  with no suffix specified this is coastline.qgs that will be choosen by default: if this is not the project you are working on then remove it from your folder.

**Comment by tarikcarli** [comment]
It does not matter to url contains coastline.qgz, coastline.qgs or coastline. I tried all of them and it didn't work. I add qgs file to make sure not to work with qgs. And also it is worth to mention it that WMS service works fine with all of urls. 
I looked out log. There is warning that it is "Logged warning: Loading a file that was saved with an older version of qgis (saved in 3.10.4-A Coruña, loaded in 3.10.6-A Coruña). Problems may occur." 
I deleted qgs file. l'll work qgz file.
